presets/TestSamples/4.3/default_rc_smoothing.txt

```diff
@@ -0,0 +1,17 @@
+# Title: Default RC smoothing
+# FirmwareVersion: 4.3
+# Category: RC_SMOOTHING
+# Official: true
+# Keywords: default rc smoothing
+# Author: Chris Thompson ctzsnooze
+# Description: Reverts RC smoothing, feedforward transition and jitter reduction to defaults
+
+set rc_smoothing = ON
+set feedforward_jitter_reduction = 10
+set feedforward_transition = 0
+set rc_smoothing_auto_factor = 30
+set rc_smoothing_auto_factor_throttle = 30
+set rc_smoothing_setpoint_cutoff = 0
+set rc_smoothing_feedforward_cutoff = 0
+set rc_smoothing_throttle_cutoff = 0
+set rc_smoothing_debug_axis = ROLL
```

Comment:
`set rc_smoothing_debug_axis = ROLL`
do we need this for user presets?

Reply:
its the default value, so if we reset to defaults, then yes